app/views/backstage/labs/edit.html.haml

```diff
@@ -51,78 +46,64 @@
 
 
     %fieldset
-      %legend Status and history of your Lab
-      .row
-        .col-lg-12
-          %h5
-            Current status of your Lab
-          %p
-            Is this Lab already open, planned or closed?
+      .text-large Status and history of your Lab
+      .text-medium
+        Current status of your Lab
+        .hint.px-0
+          Is this Lab already open, planned or closed?
 
           .row
-            - Lab::ACTIVITY_STATUS.each do |status|
-              .col-lg-2
-                .radio-inline
-                  %span
-                    = f.radio_button :activity_status, status, class: 'activity-status-radio'
-                    %label.radio{style: "width: 160px;"}
-                      = status.humanize.titleize
+          - Lab::ACTIVITY_STATUS.each do |status|
```

Comment:
Is this correct indentation? The `- Lab` is inside the row, so should it not be 2 spaces more to the right?

Reply:
yes it's correct because for each status we will have a form-check element inside col-lg-2

.row
          - Lab::ACTIVITY_STATUS.each do |status|
            .col-lg-2
              %span.form-check.mb-0.mt-3